flink-runtime/src/test/java/org/apache/flink/runtime/state/memory/ByteStreamStateHandleTest.java

```diff
@@ -172,4 +172,15 @@ public void testBulkReadINdexOutOfBounds() throws IOException {
             // expected
         }
     }
+
+    @Test
+    public void testStreamWithEmptyByteArray() throws IOException {
+        final byte[] data = new byte[0];
+        final ByteStreamStateHandle handle = new ByteStreamStateHandle("name", data);
+
+        FSDataInputStream in = handle.openInputStream();
```

Comment:
This input stream could be closed properly.

Reply:
Add try with resource when using FSDataInputStream.

Reply:
> This input stream could be closed properly.

I will add try with resource when using FSDataInputStream.